### DISCUSSION TITLE
Annoying status column toggling while navigating code

### DISCUSSION THREAD

**Discussion by cyaconi** [LazyVim/LazyVim#3251] [Q&A]
I am not sure if it is the status column where the code actions alert appears, but I am experiencing strange behavior when navigating my code. As you can see from the video, the column is toggling while I move, which is extremely annoying.
I have not found a way to fix it, could anyone assist me?
Thanks!

![sc-nvim](https://github.com/LazyVim/LazyVim/assets/91154/0b4406cc-c2e2-49c6-810e-73aac7bde854)

**Comment by folke** [comment]
I dont have that at all.
Can you please see if you can reproduce this with a stock LazyVim and if so create an issue with the exact steps to trigger it

**Reply by cyaconi** [reply]
I finally solved it by adding `vim.o.signcolumn = "yes:2"` to `options.lua`. I am not sure if this is the best solution, but given that you were unable to reproduce it, there is undoubtedly some misconfiguration in my plugins related to the new v0.10 or the slew of new updates the plugins have released in recent days.
